docs/bnb-smart-chain/developers/node_operators/full_node.md

```diff
@@ -134,3 +134,79 @@ Please read [this guide](node_maintenance.md)
 ## Upgrade Geth
 Please read [this guide](upgrade_geth.md)
 
+
+
+---
+title: Reth Node - BSC Develop
+---
```

Comment:
better divide this page into 2 part:
1.Full node by Geth client
..
1.Full node by Reth client